packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -285,6 +286,24 @@ class AndroidSdk {
     _latestVersion = _sdkVersions.isEmpty ? null : _sdkVersions.last;
   }
 
+  /// Returns the filesystem path of the Android SDK manager tool or null if not found.
+  String get sdkManagerPath {
+    final String toolPath = fs.path.join(directory, 'tools', 'bin', 'sdkmanager');
```

Comment:
What about the Windows check we had previously where we were adding `.bat`?

Reply:
You should only be executing this with processManager, which automatically figures out the correct ending.

Reply:
Right, but we'll return `null` from this method on Windows, unless I'm missing something -- we check `fs.isFileSync(toolPath)` immediately below here.